dashboard/state_aggregator.py

```diff
@@ -488,6 +492,93 @@ def sort_func(entry):
             partial_failure_warning=partial_failure_warning,
         )
 
+    async def summarize_tasks(self, option: SummaryApiOptions) -> SummaryApiResponse:
+        result = await self.list_tasks(
+            option=ListApiOptions(
+                timeout=option.timeout, limit=DEFAULT_LIMIT, filters=[]
+            )
+        )
+
+        summary = {}
+        tasks = result.result
+        for task in tasks.values():
+            if task["func_or_class_name"] not in summary:
+                summary[task["func_or_class_name"]] = {
+                    "state_counts": defaultdict(int),
+                    "type": task["type"],
+                    "func_or_class_name": task["func_or_class_name"],
+                    "required_resources": task["required_resources"],
+                }
+            task_summary = summary[task["func_or_class_name"]]
+            task_summary["state_counts"][task["scheduling_state"]] += 1
+
+        return SummaryApiResponse(
+            result=summary, partial_failure_warning=result.partial_failure_warning
+        )
+
+    async def summarize_actors(self, option: SummaryApiOptions) -> SummaryApiResponse:
+        result = await self.list_actors(
+            option=ListApiOptions(
+                timeout=option.timeout, limit=DEFAULT_LIMIT, filters=[]
+            )
+        )
+
+        summary = {}
+        actors = result.result
+        for actor in actors.values():
+            if actor["class_name"] not in summary:
+                summary[actor["class_name"]] = {
+                    "state_counts": defaultdict(int),
+                    "resource_mapping": actor["resource_mapping"],
+                }
+            task_summary = summary[actor["class_name"]]
+            task_summary["state_counts"][actor["state"]] += 1
+
+        return SummaryApiResponse(
+            result=summary, partial_failure_warning=result.partial_failure_warning
+        )
+
+    async def summarize_objects(self, option: SummaryApiOptions) -> SummaryApiResponse:
+        result = await self.list_objects(
+            option=ListApiOptions(
+                timeout=option.timeout, limit=DEFAULT_LIMIT, filters=[]
+            )
+        )
+
+        summary = {}
+        objects = result.result
+        logger.info(objects)
+        for object in objects.values():
+            if object["call_site"] not in summary:
```

Comment:
NOTE: Call_site is not recorded by default. I will add warning if the env var is not configured.